src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObj = {};
+  const cssArrey = sourceString.split(';');
+
+  cssArrey.map(function(lineCss) {
```

Comment:
you can there use callback
```suggestion
  cssArrey.map((lineCss) => {
```